dhis-2/dhis-services/dhis-service-analytics/src/main/java/org/hisp/dhis/analytics/partition/JdbcPartitionManager.java

```diff
@@ -55,6 +56,7 @@
     private Map<AnalyticsTableType, Set<String>> analyticsPartitions = new HashMap<>();
 
     @Autowired
+    @Qualifier( "analyticsJdbcTemplate" )
```

Comment:
I think this is not related to the partitioning, it would be good to keep separate branches for these fixes so that we can test, commit and potentially backport them in isolation.